build/pipelines/templates/package-msixbundle.yaml

```diff
@@ -117,10 +117,15 @@ jobs:
       targetFolder: $(Build.ArtifactStagingDirectory)\msixBundle
 
   - ${{ if eq(parameters.signBundle, true) }}:
-    - task: EsrpCodeSigning@2
+    - task: EsrpCodeSigning@5
       displayName: Send msixbundle to code signing service
       inputs:
-        ConnectedServiceName: Essential Experiences Codesign
+        ConnectedServiceName: Essential Experiences Codesign ARM
+        AppRegistrationClientId: 1c0e0438-59bc-4788-929e-f2e2d168e049
```

Comment:
We should not put this information in public. Consider retrieving the internal configuration in runtime.